client/resource_group/controller/controller.go

```diff
@@ -309,6 +336,16 @@ func (c *ResourceGroupsController) updateAvgRequestResourcePerSec() {
 }
 
 func (c *ResourceGroupsController) handleTokenBucketResponse(resp []*rmpb.TokenBucketResponse) {
+	if c.responseDeadlineCh != nil {
+		if c.run.responseDeadline.Stop() {
+			select {
+			case <-c.run.responseDeadline.C:
```

Comment:
why need this line?

Reply:
As the comment of `Timer.Stop()`, 
> Stop does not close the channel, to prevent a read from the channel succeedingincorrectly. To ensure the channel is empty after a call to Stop, check the return value and drain the channel.